src/NuGet.Core/NuGet.Commands/VerifyCommand/VerifyCommandRunner.cs

```diff
@@ -58,10 +60,16 @@ public async Task<int> ExecuteCommandAsync(VerifyArgs verifyArgs)
                         _defaultFingerprintAlgorithm)).ToList();
 
                 var verifierSettings = SignedPackageVerifierSettings.GetVerifyCommandDefaultPolicy();
+                var trustedSignersSection = verifyArgs.Settings.GetSection(TrustedSignersSectionName);
```

Comment:
This is not backwards compatible because the obsolete constructor you added will have `Settings` as null, and therefore this code will throw a `NullReferenceException`.

Perhaps have a re-read of https://github.com/NuGet/NuGet.Client/blob/dev/docs/nuget-sdk.md#types-of-compatibility as a refresher to different types of compatibility.